src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -111,7 +115,10 @@ export const formDescriptions = Object.keys(formBenefits).reduce(
     } else {
       formNumber = `(${key})`;
     }
-    const formDescription = `${formBenefits[key]} application ${formNumber}`;
+    const formDescription =
```

Comment:
just a thought: you could set the `formDescription` default value right under line 110 above and then add another `else if` to check for `key === VA_FORMS_IDS.FORM_VA_2346A` since we already have an if statement that's doing different things based on the key.

Reply:
done